src/Polymesh.ts

```diff
@@ -11,6 +11,7 @@ import {
   transferPolyX,
   TransferPolyXParams,
 } from '~/api/procedures';
+import { revokeClaims, RevokeClaimsParams } from '~/api/procedures/revokeClaims';
```

Comment:
This should be imported from `~/api/procedures`. If it's not exported from there, please add the export (my bad)